### ISSUE TITLE
Initial action and payload on addListener

### ISSUE THREAD

**Issue by lukefanning** [Neson/react-native-system-notification#18]
@Neson It seems to be the case that after creating a notification with an action and payload, the notification listener function is repeatedly being invoked on calling addListener. 

The first time you do want that behaviour as it has come from the notification that was just pressed/clicked. However on returning to that screen within the app it then calls the listener function again. I am calling addListener in componentWillMount() on a particular screen.

Looking at your code it is clear that this is happening because this.module.sysInitialNotificationPayload (and the action) is still set even after it has already been used (i.e. line 79 listener(event)).

**Comment by Neson** [comment]
Yes, it is. I'm still figuring out a better way to solve this problem. Maybe by adding an variable in the `Notification` object to log if the first registration. Or I'll go see how FB has done this on RN iOS. :smile:
